### ISSUE TITLE
Drop future support for Windows versions

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#558]
I don't think anybody is using them, so continued support is not useful.

**Comment by danlyke** [comment]
I've got at least one person who'd like to use it, but we keep having
huge performance issues when we try it on his machine. I suspect we've
got a chicken/egg problem with the Windows version...

**Comment by mpogue2** [comment]
I think I fixed those performance problems long ago, and it was his action item to re-test.  I don't think he ever did, but IIRC I left the bug open.
It's possible some people might want SquareDesk/WIN for the built-in SD, but lots of callers are quite happy with the 1975-style graphics that SD on Windows provides.
I've done a lot of Windows builds, and it's a PITA to keep up.  With no real customers (OK, maybe one *potential* customer :-), I can't see continuing to invest the time.  The one Windows customer I knew of who was using it (Nikki) moved to MacOS a while ago.

**Comment by danlyke** [comment]
Yeah, I don't get to testing very often, but last time I tested it seemed
to work well on one of his machines, and then abysmally on another. I
should see what the state of my Windows VM is so that I can do Windows
builds.

**Comment by mpogue2** [comment]
Tom Miller is interested in using SquareDesk on an HP laptop for the SD feature.  It works nice over Zoom.  Could be a reason to release again on Windows.

**Comment by mpogue2** [comment]
I think we need a maintainer for this first -- somebody who is willing to take this on.  It's not a trivial amount of work to support a platform, especially if (like me) they don't use that platform on a daily basis.
